venia-integration-tests/src/tests/integrationTests/multiStore/storeSwitching.spec.js

```diff
@@ -979,6 +979,7 @@ describe('shopping cart', () => {
         cy.wait(['@mockProduct2RouteData']);
 
         setProductColorOption('Khaki');
+        setProductSizeOption('S'); // This variant is disabled, make sure error is not thrown
```

Comment:
add assertion for no error in console and assert product was disabled

Reply:
I added the assertion to make sure the add to cart button showed "OUT OF STOCK", which is the desired behavior if the selected option is disabled.

Reply:
The error that is caused is an unhandled exception, which did not get caught when I spied on console.error. There is a way for me to catch an unhandled exception, detailed in the cypress docs: https://docs.cypress.io/api/events/catalog-of-events#To-conditionally-turn-off-uncaught-exception-handling-unhandled-promise-rejections

I feel like it would get kind of hacky for me to create an assertion out of this, and seems unnecessary since an unhandled exception should fail the test by default. Do you have any ideas on how I should proceed?

Reply:
That will work. Just need a way to ensure it behaves correctly.